src/transformers/training_args.py

```diff
@@ -93,6 +93,15 @@ def get_int_from_env(env_keys, default):
     return default
 
 
+def get_xla_device_type(device: torch.device) -> Optional[str]:
```

Comment:
```suggestion
def get_xla_device_type(device: "torch.device") -> Optional[str]:
```
We need a forward reference here as `torch` isn't necessarily imported in this file.

Reply:
Thanks!